README.md

```diff
@@ -8,6 +8,11 @@
   <img alt="Screenshot" title="cmp-nvim-ultisnips" src="screenshots/preview.png" width="80%" height="80%">
 </p>
 
+## Features
+- **Composable Mappings**: get rid of boilerplate code in your config
+- **Customization**: change how your snippets are displayed by cmp
+- **Treesitter Integration**: show snippets based on your current position in a file
+
 ## Installation and Recommended Mappings
```

Comment:
Please update README to provide the option we just offered: treesitter integration and the default value